infrastructure/infrastructure-impl/src/main/java/eu/f4sten/infra/impl/InfraConfig.java

```diff
@@ -100,6 +100,25 @@ public PostgresConnector bindPostgresConnector() {
         return new PostgresConnectorImpl(args.dbUrl, args.dbUser, true);
     }
 
+    @Provides
+    @Singleton
+    public KafkaConnector bindKafkaConnector(RunnerArgs runnerArgs) {
+        assertFor(args) //
+                .notNull(a -> a.kafkaUrl, "kafka url") //
+                .that(a -> a.instanceId == null || !a.instanceId.isEmpty(), "instance id must be null or non-empty") //
+                .that(a -> a.kafkaGroupId == null || !a.kafkaGroupId.isEmpty(), "group id must be null or non-empty");
+
+        var pluginId = runnerArgs.run.replace("eu.f4sten.", "");
```

Comment:
Why is `eu.f4sten.` replaced with an empty string?

Reply:
This is a simple means to make the logging easier to read. Instead of the full "eu.fasten.mavencrawler.Main", I cut off the `eu.fasten` part (And afaik also the `Main`). This is nto new functionality though, this has been there before.